File: public/e-commerce/products/import streamlit as st.r
import streamlit as st
import pandas as pd
import plotly.express as px

st.title("📊 Simple Dashboard")

# Sample data
data = {
    "Days": ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"],
    "Sales": [200, 300, 400, 350, 500, 600, 550]
}
df = pd.DataFrame(data)

# Show simple cards
st.metric("Total Sales", "₹2,900")
st.metric("Growth", "12.5%")

# Simple chart
fig = px.line(df, x="Days", y="Sales", title="Weekly Sales Chart")
st.plotly_chart(fig)

File: public/e-commerce/products/import streamlit as st.py
import streamlit as st

# Title
st.title("📊 My Simple Dashboard")

# Create some cards using columns
col1, col2, col3 = st.columns(3)

with col1:
    st.header("Sales")
    st.write("₹25,000")

with col2:
    st.header("Profit")
    st.write("₹5,000")

with col3:
    st.header("Growth")
    st.write("20%")

# Simple chart using built-in method
st.subheader("Sales Trend")
st.line_chart([200, 300, 400, 350, 500, 450, 600])